### PR TITLE
Clean pylint baseline

### PR DESCRIPTION
**Pull request by cchristelis**

### CODE REVIEW

File: realtime/shake_event.py [hunk 1]
```diff
@@ -803,18 +803,13 @@ def sorted_impacted_cities(self, row_count=5):
...
-                               d['roman'],
-                               d['dist_to'],
-                               d['dir_to'],
-                               d['dir_from'],
-                               d['id']))
+        sorted_cities = sorted(
+            cities,
+            key=lambda d: (
+                # we want to use whole no's for sort
+                -d['mmi-int'], -d['population'], d['name'],
```
Comment (timlinux) [inline realtime/shake_event.py]: can you keep this one item per line as per our 'pull left' policy (if items dont fit on one line they should be placed one item per line)

File: safe/common/tables.py [hunk 2]
```diff
@@ -201,7 +202,8 @@ class TableRow(object):
     Reference: http://www.w3.org/tr/html4/struct/tables.html#h-11.2.5
     """
 
-    def __init__(self, cells=None, bgcolor=None, header=False, attribs=None,
+    def __init__(
+            self, cells=None, bgcolor=None, header=False, attribs=None,
```
Comment (timlinux) [inline safe/common/tables.py]: Again one item per line policy would apply here, though this is third party code so not as critical

File: safe/common/test/test_tables.py [hunk 3]
```diff
@@ -248,13 +248,11 @@ def test_col_span(self):
...
         table_cell_aa = TableCell('aa spanned', row_span=2)
-        table_row1 = TableRow([table_cell_aa,
-                              self.table_cell_b,
-                              self.table_cell_c,
-                              self.table_cell_d])
-        table_row2 = TableRow([self.table_cell_b,
-                               self.table_cell_c,
-                               self.table_cell_d])
+        table_row1 = TableRow([
+            table_cell_aa, self.table_cell_b, self.table_cell_c,
```
Comment (timlinux) [inline safe/common/test/test_tables.py]: one item per line

File: safe/impact_functions/volcanic/volcano_building_impact.py [hunk 4]
```diff
@@ -279,15 +279,17 @@ def run(self, layers):
             # Count affected buildings by usage type if available
             usage = None
             for building_type_attribute in building_type_attributes:
-                if building_type_attribute in attribute_names_lower and (
-                                usage is None or usage == 0):
+                if (
```
Comment (timlinux) [inline safe/impact_functions/volcanic/volcano_building_impact.py]: Wow does it really need a new line? I never figured a way to format multiline if statements nicely.

**Comment by timlinux** [comment]
Thanks @cchristelis LGTM once tests are passing and minor comments above taken into account go ahead and merge.

test this please, jenkins

File: safe/impact_functions/volcanic/volcano_building_impact.py (hunk 4 above)
Comment (cchristelis) [inline safe/impact_functions/volcanic/volcano_building_impact.py]: So this is closest to our pull left policy. Also it has one bit of logic per line, which is close to the one item per line policy as requested above. 
Do you think we should prefer a different convention for logical statements?

**Pull request merged**